libr/core/cbin.c

```diff
@@ -1015,6 +1015,41 @@ static int bin_info(RCore *r, int mode, ut64 laddr) {
 	return true;
 }
 
+typedef struct {
+	int *line_starts;
```

Comment:
size_t

Reply:
That also means changing the signature of r_str_split_lines and all the code using it. I can do it if you want, there aren't too many such places.

Reply:
Yes i understand, just trying to be more formal and correct on every PR we merge. Not sure what @ret2libc thinks about that

Reply:
Refactored in separate commit.